packages/svelte/src/internal/client/dom/elements/events.js

```diff
@@ -1,6 +1,34 @@
 import { render_effect } from '../../reactivity/effects.js';
 import { all_registered_events, root_event_handles } from '../../render.js';
 import { define_property, is_array } from '../../utils.js';
+import { hydrating } from '../hydration.js';
+
+/**
+ * SSR adds onload and onerror attributes to catch those events before the hydration.
+ * This function detects those cases, removes the attributes and replays the events.
+ * @param {HTMLElement} dom
+ */
+export function replay_events(dom) {
+	if (hydrating) {
```

Comment:
we can early-return to get rid of a level of indentation

```suggestion
	if (!hydrating) return;
```